cmd/podman/main.go

```diff
@@ -138,6 +138,17 @@ func main() {
 			logrus.SetLevel(level)
 		}
 
+		// Only if not rootless, set rlimits for open files.
+		// We open numerous FDs for ports opened
+		if os.Geteuid() == 0 {
+			rlimits := new(syscall.Rlimit)
+			rlimits.Cur = 1048576
+			rlimits.Max = 1048576
```

Comment:
I don't think so, but I'll ask anyway.  Is this a value we should tuck away in a conf file?